### ISSUE TITLE
Unable to obtain custom Pinterest scope

### ISSUE THREAD

**Issue by dzbrozek** [cuongdevjs/reactjs-social-login#126]
Custom scope is always overwritten by default scope:

```
const oauthUrl = `${PINTEREST_URL}/?client_id=${client_id}&scope=${scope}&state=${
   state + '_pinterest'
}&redirect_uri=${redirect_uri}&response_type=code&scope=boards:read,pins:read,user_accounts:read
```

https://github.com/cuongdevjs/reactjs-social-login/blob/03578acb5b4b39ee10d4b0231bff4cae8f693c54/src/LoginSocialPinterest/index.tsx#L143

**Comment by cuongdevjs** [comment]
Thanks for your report. I gonna take a look it now.

**Comment by cuongdevjs** [comment]
This issue has been fixed, please upgrade to version 2.6.3. Thank you so much.
